### DISCUSSION TITLE
Perenio PECMS01

### DISCUSSION THREAD

**Discussion by Asmcavr** [Koenkk/zigbee2mqtt#11028] [Devices]
The battery level is displayed incorrectly, more than 100% in my case 180%

**Comment by Koenkk** [comment]
Fixed now, after updating make sure to wait at least 1 day to make sure a new battery value report comes in.

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Reply by Asmcavr** [reply]
Thank you very much. When can we expect a fix in the release version?

**Reply by Koenkk** [reply]
1 February
